src/shell/main.cpp

```diff
@@ -474,6 +474,15 @@ static command_executor commands[] = {
         "<-a --app_name str> [-f --forced]",
         cancel_bulk_load,
     },
+    {
+        "detect_hotkey",
+        "start or stop hotkey detection on the replica",
+        "[-a|--app_name str]"
+        "[-p|--partition_index num]"
+        "[-t|--hotkey_type read|write]"
+        "[-c|--detect_action start|stop]",
```

Comment:
`<>` to show required parameters, and `[]` is for optional parameters. I suggest using `<>`.